seqr/views/utils/export_utils_2_3_tests.py

```diff
@@ -85,7 +88,20 @@ def mock_write_zip(file, content):
         })
         mock_zip_content = {}
 
+        # test unicode chars in zip filename
+        response = export_multiple_files(
+            [['file1', header1, rows], ['file2', header2, rows]], 'zipfilen\xe2me', file_format='tsv')
+        self.assertEqual(response.status_code, 200)
+        filename = response.get('content-disposition')
+        self.assertEqual(filename, 'attachment; filename="zipfilen\xc3\xa2me.zip"' if isinstance(filename, str)
+        else b'attachment; filename="zipfilen\xc3\xa2me.zip"')
```

Comment:
so in general conditional logic in tests is not good, and python 3 doesn't even support `b'...'` notation so I really don't think this is a good way to test. Couldn't you cast it to bytes before checking equality?

Reply:
Python 3 doesn't support `u''` for unicode literals but it does support `b''` for bytes literals. I can change it to casting.